pkg/pillar/cmd/volumemgr/handlecontent.go

```diff
@@ -277,3 +292,25 @@ func deleteContentTree(ctx *volumemgrContext, status *types.ContentTreeStatus) {
 	deleteLatchContentTreeHash(ctx, status.ContentID, uint32(status.GenerationCounter))
 	log.Infof("deleteContentTree for %v Done", status.ContentID)
 }
+
+// lookupDatastoreConfig look for a datastore config
+func lookupDatastoreConfig(ctx *volumemgrContext, dsID uuid.UUID,
```

Comment:
Why return an error string and not use the go error type?

Reply:
No idea, honestly. I just took it from downloader as is. I can make it better; your suggestion makes sense.

Reply:
I extracted it out to `utils`. There may be a larger pattern here, but let's start with simple. This PR has enough in it to begin with.